pysilcam/silcamgui/silcamgui.py

```diff
@@ -257,12 +255,12 @@ def fillInConfigEditor(self, inputFile):
         self.ui.com_portEdit.setCurrentIndex(0)
 
         for port in listPortCom:
-            if (port != self.settings.PostProcess.com_port):
+            if port != self.settings.PostProcess.com_port:
                 self.ui.com_portEdit.addItem(port)
 
         self.ui.window_sizeEdit.setText(str(self.settings.PostProcess.window_size))
 
-        if (self.settings.ExportParticles.export_images == True):
+        if self.settings.ExportParticles.export_images == True:
```

Comment:
E712: comparison to True should be 'if cond is True:' or 'if cond:'  <!-- Automatically posted by Lintly -->